docs/output.md

```diff
@@ -404,11 +405,32 @@ Besides the reference files or output files created by BUSCO, the following summ
 
 </details>
 
+### Bin Refinement
+
+[DAS Tool](https://github.com/cmks/DAS_Tool) is an automated method that integrates the results of a flexible number of binning algorithms to calculate an optimized, non-redundant set of bins from a single assembly. nf-core/mag uses this tool to attempt to further improve bins based on combining the MetaBAT2 and MaxBin2 binning output, assuming sufficient quality is met for those bins.
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `GenomeBinningRefinement/`
+  - `bin_depths_summary_refined.tsv`: Summary of bin sequencing depths for all samples against refined for refined bin contigs. Depths are available for samples mapped against the corresponding assembly, i.e. according to the mapping strategy specified with `--binning_map_mode`. Only for short reads
+  - `bin_summary.tsv`: Summary of bin and refined bin sequencing depths together with BUSCO, QUAST and GTDB-Tk results, if at least one of the later was generated. This will be alternatively placed in `GenomeBinning/` when **no** binning refinement is performed
+  - `[assembler]-[sample/group]_allBins.eval`: Tab-delimited description with quality and completeness metrics for the input bin sets
```

Comment:
Could we add a few words how this is computed or where this is retrieved from? For example:
```suggestion
  - `[assembler]-[sample/group]_allBins.eval`: Tab-delimited description with quality and completeness metrics for the input bin sets.  Quality and completeness are estimated by DAS_TOOL using a scoring function based on the frequency of 51 bacterial or 38 archaeal reference single-copy genes (SCG).
```

Reply:
I actually copy/pasted directly from the DAS_Tool docs... but what you've written is a good summary.